Makefile

```diff
@@ -1,7 +1,7 @@
-CC=clang++
+CC=g++
```

Comment:
Drive by comment: I don't think changing the compiler is necessary. There don't seem to be g++ specific extensions or peculiarities being used anywhere. Perhaps whomever wants to compile with (not clang++) can be specified as a environment variable? (à la make -CC=g++). <3 ltns, hope you're doing well Ray

Reply:
Nice catch @iajrz. I didn't mean to commit that particular change. I changed the compiler locally since I didn't have clang++ installed on my system.  I'll update the Pull Request to remove this compiler change.

Reply:
Done @iajrz . I've set the compiler back to clang++.